CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ see if you need to change any of these:
   to star PEcAn with https.
 - TRAEFIK_HTTPS_OPTIONS is no longer used, this is the default when you use https.
 - TRAEFIK_HTTP_REDIRECT is no longer used, this is the default when you use https.
+- Updated `soil_process` function in `soil_process.R` and `ic_process` to use `PEcAn.DB::query.site` instead of `PEcAn.data.atmosphere::db.site.lat.lon` for retrieving site latitude and longitude.
```

Comment:
This change won't be visible to end users, so it doesn't really need a line in the PEcAn changelog. 
Instead, please add an entry in the NEWS.md of data.land stating that  data.land no longer depends on data.atmosphere, including a reference to this PR number and a credit to yourself for the change.